test_scripts/API/ButtonSubscription/031_SubscribeButton_on_different_buttons_Unexpected_disconect.lua

```diff
@@ -0,0 +1,67 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0192-button_subscription_response_from_hmi.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) Mobile app is subscribed for button_1 and button_2
+-- 2) Unexpected disconnect and connect are performed
+-- 3) App registers with actual hashId
+-- 4) SDL sends Buttons.SubscribeButton(button_1, appId) to HMI during resumption
+-- 5) SDL sends Buttons.SubscribeButton(button_2, appId) to HMI during resumption
+-- 6) HMI responds with error code to SubscribeButton(button_1, appId)
```

Comment:
Add info that HMI responds with success to another request in current step or in separate one